src/main/java/com/xebialabs/overthere/cifs/winrm/KerberosJaasConfiguration.java

```diff
@@ -49,20 +49,26 @@
             options.put("debug", "true");
         }
         
-        if (ticketCache) {
-        	options.put("useTicketCache", "true");
-        } else {
-        	options.put("useTicketCache", "false");
-        }
-        
         options.put("refreshKrb5Config", "true");
         
         if (JavaVendor.isIBM()) {
             options.put("credsType", "initiator");
+            
+            if (ticketCache) {
+            	options.put("useDefaultCcache", "true");
```

Comment:
Yes, I did mean `useDefaultCcache`. We found the distinction in [this IBM document](http://www-01.ibm.com/support/knowledgecenter/SS69YH_6.0.0/com.ibm.spss.cads.config.doc/model_management/thin/jboss_sso.html?lang=en).